pkg/util/etcdutil/etcdutil.go

```diff
@@ -37,3 +37,20 @@ func ListMembers(endpoints []string) (*clientv3.MemberListResponse, error) {
 	etcdcli.Close()
 	return resp, err
 }
+
+func RemoveMember(clientURLs []string, id uint64) error {
```

Comment:
this is def what we should do. can you separate this into another pr and get it merged first?